charts/stardog/README.md

```diff
@@ -40,7 +40,7 @@ Configuration Parameters
 | `persistence.size`                           | The size of volume for Stardog home |
 | `ports.server`                               | The port to expose Stardog server |
 | `ports.sql`                                  | The port to expose Stardog BI server |
-| `tmpDir`                                     | The directory to use for Stardog tmp space |
+| `tmpDir`                                     | The directory to use for Stardog tmp space. If you choose to place this in STARDOG_HOME (/var/opt/stardog) for performance reasons, insure that it does not conflict with any possible database names. For example, a good choice might be /var/opt/stardog/tmp-4646E7B662A7. If the directory does not exist it will be created. |
```

Comment:
s/insure/ensure

Reply:
I'm going to switch to ensure. But I've also been told by American's to use insure when I'm looking to mean "to make certain" under the false belief that ensure is British spelling. This is the trouble when you come from America's hat :)

> The Penguin Writer’s Manual notes that the verb “generally used in the active form to mean “make (something) certain” is ensure,” but also notes that ensure is often spelled insure in American English. 

https://www.dailywritingtips.com/insure-vs-ensure/